### PR TITLE
AM-6339 Update kuebslice-cli release version

### PR DESCRIPTION
**Pull request by uma-kt**
- Updated the release version.
- Revised the content, wherever applicable.

### CODE REVIEW

**Review by priyank-upadhyay — APPROVED** [review]

File: versioned_docs/version-0.5.0/kubeslice-cli/installing-kubeslice.mdx [hunk 1]
```diff
@@ -33,42 +33,11 @@ kubeslice-cli --config=<path-to-custom-topology-file> install
...
-      name: controller
-      context_name: kind-controller
-    workers:
-      - name: worker-3
-        context_name: ks-w-3      
-  kubeslice_configuration:
-    project_name: demo
-  ```
+The kubeslice-cli allows you to add a new worker cluster to an existing KubeSlice configuration. Add new worker cluster information under workers
+list in the custom topology file that was used to install KubeSlice earlier and using **install** command, register the new worker
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.5.0/kubeslice-cli/installing-kubeslice.mdx:39]: the install command

File: versioned_docs/version-0.5.0/kubeslice-cli/installing-kubeslice.mdx [hunk 2]
```diff
@@ -77,22 +46,21 @@ cluster. The `-s enterprise` option skips using enterprise installation, while t
...
+## Create a Slice
 
 To onboard your existing namespaces (and their applications) onto a slice, follow these steps:
 
 1. Create a slice configuration YAML file (choose the namespaces, clusters, and so on to be part of the slice).
 2. Use the `kubeslice-cli create sliceConfig` command to apply the slice configuration YAML file.
 
-
-### Create a Slice
+### Creating the Slice Configuration YAML File
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.5.0/kubeslice-cli/installing-kubeslice.mdx:63]: Creating a Slice Configuration ...

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx [hunk 3]
```diff
@@ -60,16 +60,15 @@ kubeslice-cli --config=<topology-configuration-file> install
 The above command installs the KubeSlice Controller, creates a project, and registers the worker cluster with the project by 
 installing the Slice Operator on the worker cluster.
 
-## Register a Worker Cluster
+## Register a New Worker Cluster
 
-The kubeslice-cli allows you to add a new worker cluster to an existing KubeSlice configuration.
+To register a new worker cluster with the existing KubeSlice configuration (or KubeSlice Controller), follow these steps:
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx:65]: follow these steps is redundant. 

the sentence can end with a colon with (or the KubeSlice Controller):

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx [hunk 4]
```diff
@@ -60,16 +60,15 @@ kubeslice-cli --config=<topology-configuration-file> install
...
 
-:::note
-Add new worker cluster information to the custom topology file that was used to install KubeSlice earlier and 
-using **install** command, register the new worker cluster with the KubeSlice controller.
-:::
+1. Add new worker cluster information under workers in the custom topology file that was used to install KubeSlice earlier.
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx:67]: make workers as a property in the template as under `workers`

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx [hunk 5]
```diff
@@ -60,16 +60,15 @@ kubeslice-cli --config=<topology-configuration-file> install
...
+2. Use the **install** command to apply the updated custom topology file.
 
-The following is an example custom topology file for registering a new worker cluster.
+The following is an example custom topology file for registering a new worker cluster. Under worker, add a
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx:70]: Add backticks for worker as in `worker`

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx [hunk 6]
```diff
@@ -104,15 +103,15 @@ Use the following command to register a new worker cluster with the KubeSlice Co
 kubeslice-cli install --config=<new-worker-topology-yaml> -s controller
 ```
 
-## Onboard Namespaces
+## Create a Slice
 
 To onboard your existing namespaces (and their applications) onto a slice, follow these steps:
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx:108]: onto a slice: 

follow these steps is redundant

**Review by veenamj-avesha — APPROVED** [review]
Left my feedback to incorporate them before merging. Approving the other changes.

File: versioned_docs/version-0.5.0/kubeslice-cli/installing-kubeslice.mdx (hunk 1 above)
Comment (uma-kt) [reply versioned_docs/version-0.5.0/kubeslice-cli/installing-kubeslice.mdx]: updated

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx (hunk 3 above)
Comment (uma-kt) [reply versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx]: done

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx (hunk 4 above)
Comment (uma-kt) [reply versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx:67]: done

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx (hunk 5 above)
Comment (uma-kt) [reply versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx]: done

File: versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx (hunk 6 above)
Comment (uma-kt) [reply versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx:108]: done

**Pull request merged**